src/script/builder/tir/prim_func_frame.h

```diff
@@ -33,10 +33,10 @@ class PrimFuncFrameNode : public TIRFrameNode {
   Optional<String> name;
   Array<tvm::tir::Var> args;
   Optional<Type> ret_type;
-  Map<tvm::tir::Var, Buffer> buffer_map;
-  Map<tvm::tir::Var, Buffer> preflattened_buffer_map;
+  Map<tvm::tir::Var, tvm::tir::Buffer> buffer_map;
+  Map<tvm::tir::Var, tvm::tir::Buffer> preflattened_buffer_map;
   Map<String, ObjectRef> attrs;
-  BlockFrame root_block_frame{nullptr};
+  Array<tvm::tir::Buffer> alloc_buffers;
```

Comment:
use a more informative name

```suggestion
  Array<tvm::tir::Buffer> root_alloc_buffers;
```